runtime/darwinia/src/lib.rs

```diff
@@ -604,6 +604,88 @@ impl darwinia_treasury::Trait for Runtime {
 	type WeightInfo = weights::darwinia_treasury::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const LaunchPeriod: BlockNumber = 28 * DAYS;
+	pub const VotingPeriod: BlockNumber = 28 * DAYS;
+	pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
```

Comment:
@AurevoirXavier should we keep the parameter the same as Polkadot, I remember is `3 DAYS`

Reply:
It's 3 hours. https://github.com/paritytech/polkadot/blob/v0.8.26/runtime/polkadot/src/lib.rs#L409